.github/workflows/rebuild-changelog.yaml

```diff
@@ -33,16 +33,19 @@ jobs:
             tagName=$(echo $line | awk '{ print $2 }')
             echo "line: $line --- tag: $tagName"
 
-            json=$(gh release view $tagName --json=name,publishedAt,url,body)
-            url=$(echo $json | jq -r .url)
-            name=$(echo $json | jq -r .name)
-            date=$(echo $json | jq -r .publishedAt | cut -f1 -dT)
-            body=$(echo $json | jq -r .body)
-            echo "## [$name]($url) - $date" >> CHANGELOG.md
-            echo "" >> CHANGELOG.md
-            echo "$body" >> CHANGELOG.md
-            echo "" >> CHANGELOG.md
-            echo "" >> CHANGELOG.md
+            if [ "$tagName" != "Pre-release" ] && [ "$tagName" != "Preview" ]; then
```

Comment:
We were getting `release not found` with the github cli for releases with these tag names. I think we probably don't want to include them in the changelog anyway.

Reply:
Updated those releases, also cleaned up their names. Removed that line 👍